src/filerepository.js

```diff
@@ -0,0 +1,455 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module upload/filerepository
+ */
+
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+
+import CKEditorError from '@ckeditor/ckeditor5-utils/src/ckeditorerror';
+import ObservableMixin from '@ckeditor/ckeditor5-utils/src/observablemixin';
+import Collection from '@ckeditor/ckeditor5-utils/src/collection';
+import mix from '@ckeditor/ckeditor5-utils/src/mix';
+import log from '@ckeditor/ckeditor5-utils/src/log';
+
+import FileReader from './filereader.js';
+
+import uid from '@ckeditor/ckeditor5-utils/src/uid.js';
+
+/**
+ * FileRepository plugin.
+ *
+ * @extends module:core/plugin~Plugin
+ */
+export default class FileRepository extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	static get pluginName() {
+		return 'fileRepository';
```

Comment:
But at the same time, you have 'bold' instead of 'basicstyles/bold' in the toolbar. What's the difference?